CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ### Master
 
+ * Fix ActiveModelSerializer serializePolymorphicType and typeForRoot methods for properly serializing and extracting namespaced model names in polymorphic type attributes.
```

Comment:
Can you mention the issue name, and also just quickly say it's now accepting `::` instead of `/`